lib/charms/observability_libs/v1/cert_handler.py

```diff
@@ -638,3 +639,6 @@ def _check_juju_supports_secrets(self) -> bool:
             logger.debug(msg)
             return False
         return True
+
+    def _stable_hash(self, data):
```

Comment:
Imho this doesn't need to be a separate method. I vote we put this logic directly in `_csr_hash`, or if you feel like it should be its own function, either make it a staticmethod or make it a module-level helper function. No need to bind it to self.